### ISSUE TITLE
Site CSS makes it less usable on ultrawide monitors

### ISSUE THREAD

**Issue by EthanStandel** [docura/docura#1]
Reference issue: https://github.com/learning-rust/site/issues/62

**Comment by EthanStandel** [comment]
I opened a PR if you're interested

https://github.com/docura/docura/pull/2/files

**Comment by dumindu** [comment]
Hi @EthanStandel ,
Thanks for your feedback. 

- Actually I tested these custom zooms on a QHD monitor; I thought it's better adding a custom zoom especially for the devices like TVs.
- However I added these in a rush and recently saw this can conflict with `#article->width: 66.64vw` as well.

![Screen Shot 2022-10-18 at 12 16 07 AM](https://user-images.githubusercontent.com/1280428/196232373-018f4b65-1ca7-41e0-9644-29fd64d87018.png)

So I also think it's better remove these custom zooms for the moment.

Thanks again for the feedback :)
